filament/src/RendererUtils.cpp

```diff
@@ -92,9 +94,15 @@ FrameGraphId<FrameGraphTexture> RendererUtils::colorPass(
                 const bool canResolveDepth = engine.getDriverApi().isAutoDepthResolveSupported();
 
                 if (!data.depth) {
-                    // clear newly allocated depth buffers, regardless of given clear flags
+                    // clear newly allocated depth/stencil buffers, regardless of given clear flags
                     clearDepthFlags = TargetBufferFlags::DEPTH;
-                    data.depth = builder.createTexture("Depth Buffer", {
+                    clearStencilFlags = config.enabledStencilBuffer ?
+                            TargetBufferFlags::STENCIL : TargetBufferFlags::NONE;
+                    const char* const name = config.enabledStencilBuffer ?
+                             "Depth/Stencil Buffer" : "Depth Buffer";
+                    TextureFormat format = config.enabledStencilBuffer ?
+                            TextureFormat::DEPTH32F_STENCIL8 : TextureFormat::DEPTH32F;
+                    data.depth = builder.createTexture(name, {
```

Comment:
make sure this works with transmission, which reuses the depth buffer (also transmission + MSAA)

Reply:
Good idea, I'll check